### DISCUSSION TITLE
Extend config to set facingDirection

### DISCUSSION THREAD

**Discussion by neu5** [Annoraaq/grid-engine#57] [Ideas]
Currently while defining config of characters it's impossible to set starting facing direction. It's hardcover as 'down'.
How about adding facingDirection option in config so you can set it up from the start instead of eg. using turnTowards method?

**Comment by Annoraaq** [comment]
Great idea. I will create an issue

**Reply by Annoraaq** [reply]
Will be in the upcoming 1.10 release
